src/app/pages/StakingPage/Features/CommissionBounds/index.tsx

```diff
@@ -61,7 +61,11 @@ export const CommissionBounds = memo((props: Props) => {
       .slice()
       .sort((a, b) => a.epochStart - b.epochStart)
       .map((b, i) => <CommissionBound bound={b} key={i} />)
-    return <>{items}</>
+    return (
+      <Box as="ul" margin="none" pad="none">
```

Comment:
Is using `Box` the correct thing here or should this be `List` (https://v2.grommet.io/list)? Also not sure about setting margin and pad to `none` instead of using a proper class?

Reply:
I didn't use List component, because it's an overkill for such a case to render simple HTML markup like <ul><li> https://github.com/grommet/grommet/blob/master/src/js/components/List/List.js. It has build in InfiniteScroll etc.

Anyway, added a commit with List if that's our preference here. For no using classes, that's how Grommet API works.

Reply:
Fair point. Feel free to use the original commit. Thanks.